src/github.com/getlantern/chained/dialer.go

```diff
@@ -88,7 +89,7 @@ func checkCONNECTResponse(r *bufio.Reader, req *http.Request) error {
 	if err != nil {
 		return fmt.Errorf("Error reading CONNECT response: %s", err)
 	}
-	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+	if resp.StatusCode < http.StatusOK || resp.StatusCode > 299 {
```

Comment:
If you consider `200` a magic number here you may want to say go`http.StatusOK+99` instead of `299`.